google-cloud-logging/src/main/java/com/google/cloud/logging/Instrumentation.java

```diff
@@ -37,7 +37,8 @@ public final class Instrumentation {
   public static final String INSTRUMENTATION_NAME_KEY = "name";
   public static final String INSTRUMENTATION_VERSION_KEY = "version";
   public static final String JAVA_LIBRARY_NAME_PREFIX = "java";
-  public static final String DEFAULT_INSTRUMENTATION_VERSION = "UNKNOWN";
+  public static final String DEFAULT_INSTRUMENTATION_VERSION =
+      "1.0.0"; // {x-release-please-version}
```

Comment:
You probably want to use the java-specific replacement annotation: https://github.com/googleapis/java-logging/blob/b9e5824db9a294a898b0e84f26212300430c6526/google-cloud-logging/pom.xml#L6

Reply:
Added an update - can you please let me know if thats the right way?